index/indexer.go

```diff
@@ -204,35 +235,71 @@ func (e *indexer) index(pkgs []*packages.Package) (*Stats, error) {
 	}, nil
 }
 
-func (e *indexer) indexPkg(p *packages.Package, proID string) (err error) {
-	log.Infoln("Package:", p.Name)
+func (e *indexer) indexPkgDocs(p *packages.Package, proID string) (err error) {
+	log.Infoln("Emitting documents for package", p.Name)
 	defer log.Infoln()
 
 	for _, f := range p.Syntax {
 		fpos := p.Fset.Position(f.Package)
+		if !strings.HasPrefix(fpos.Filename, e.projectRoot) {
```

Comment:
Not a blocker, but feels like a regression? https://github.com/sourcegraph/lsif-go/issues/27

Reply:
Should have commented on the next line:

>// Indexing test files means that we're also indexing the code generated by go test;

Reply:
These files won't ever be served by the LSIF backend. I will verify by hand that test files still have support (that was originally enabled by setting `Test:true` when loading the package).

Reply:
Maybe just the comment itself confused me 😁

Reply:
Still indexes test files.